text/0000-maturity-of-otel.md

```diff
@@ -0,0 +1,149 @@
+# Framework for defining the maturity of OpenTelemetry and its SIGs
+
+On 08 Mar 2023, the OpenTelemetry GC and TC held an OpenTelemetry Leadership summit, discussing various topics. One of the themes we discussed was establishing standard rules for describing the maturity of the OpenTelemetry project. This OTEP summarizes what was discussed there and is intended to have the wider community provide feedback.
+
+This OTEP builds on what was previously communicated by the project, especially on the following page: https://opentelemetry.io/docs/reference/specification/versioning-and-stability.
+
+The Collector’s [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
+
+## Motivation
+
+Quite often, the community is faced with the question of the quality and maturity expectations of its diverse set of components. This OTEP aims to bring clarity by establishing a framework to communicate the project's maturity. As the OpenTelemetry project comprises a multitude of SIGs, and each SIG has several components of varying quality, having this framework will help set the right expectations for OpenTelemetry users.
+
+Only once we reach the “stable” level at the project level will we be ready to consider graduating OpenTelemetry at the CNCF, with all of the Tier-1 SIGs being in the context of the graduation, including their Tier-1 Components. This OTEP provides a framework for describing project maturity as a whole.
+
+## Glossary
+
+* *SIG:* an official SIG within the OpenTelemetry project, like the Collector or Java. Some WGs are temporarily accepted, given they should be SIGs instead of WGs, like the Semantic Conventions WG.
+* *Tier-1 SIG:* a SIG within the OpenTelemetry project deemed critical by the GC/TC.
+* *Candidate Tier-1 SIG:* a SIG that is deemed as critical for the future of the project, but might not be of the same maturity level yet. It’s expected to reach that level eventually, and other Tier-1 SIGs should be prepared to accommodate the new component as needed.
+* *Component:* module maintained by a SIG like a Collector exporter or an SDK sampler.
```

Comment:
Do we consider making this more general (e.g. "Artifact") so we can cover things that are not considered as "module" in common sense? For example, OpenMetrics compatibility specification, OTLP JSON format, some guidance documents.

Reply:
I think a component is more generic than an artifact. Perhaps "deliverable"? Or even component, but changing the "module" part of the description?